htsget-http-core/src/async_http_core.rs

```diff
@@ -1,3 +1,4 @@
+use log::trace;
```

Comment:
Since aws_rust_lambda_runtime uses [tracing](https://crates.io/crates/tracing) as logger (i.e https://github.com/umccr/s3-rust-noodles-bam/blob/master/Cargo.toml#L18), I wonder if it would make sense to consolidate now and avoid logging sink/consumer fragmentation in the near future.

Reply:
I think I left this one in accidentally.

However, it raises a good question. Currently we aren't using any logging. Do you think we should implement log messages like debug, info, and warn into relevant areas of the code?

Reply:
In my experience is definitely a nice to have, but since we have a few things in flux, let's leave it as a per-need basis when we are running/evaluating our code in AWS proper.